form/create_custom_field_type.rst

```diff
@@ -144,13 +144,23 @@ These are the most important methods that a form type class can define:
 
 ``configureOptions()``
     It defines the options configurable when using the form type, which are also
-    the options that can be used in ``buildForm()`` and ``buildView()`` methods.
+    the options that can be used in ``buildForm()`` and ``buildView()``
+    methods. Options are inherited from parent types and parent type
+    extensions, but you can create any custom option you need.
 
 ``finishView()``
     When creating a form type that consists of many fields, this method allows
     to modify the "view" of any of those fields. For any other use case, it's
     recommended to use instead the ``buildView()`` method.
 
+``getParent()``
```

Comment:
In this explanation I miss the "why/when use his". Also, I'd move (and merge) the above paragraph that mentions the `getParent()` method. The final result could look something like this:

```rst
``getParent()``
    If your custom type is based on another type (i.e. they share some functionality)
    add this method to return the fully-qualified class name of that original type.
    Symfony will call all the form type methods (``buildForm()``, ``buildView()``, etc.)
    of the parent type and it will execute all its type extensions before calling
    the ones defined in your custom type.

    When your custom type doesn't extend from another type, there's no need to
    implement this method because Symfony will make your type extend from the
    generic :class:`Symfony\\Component\\Form\\Extension\\Core\\Type\\FormType`
    automatically.
```

Reply:
I agree! Sorry, I had seen your comment on the original PR but forgot to apply it after the rebase.

I've also slightly reworded the paragraph above.